docs/configuration.md

```diff
@@ -392,6 +392,16 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td>Int.MaxValue</td>
```

Comment:
formatting is wrong, you lost the first column of maxReqsInFlight
 Add a line note